utils/kernel.c

```diff
@@ -467,6 +465,46 @@ static int __setup_kernel_tracing(struct uftrace_kernel_writer *kernel)
 	return -EINVAL;
 }
 
+static void skip_kernel_functions(struct uftrace_kernel_writer *kernel)
+{
+	unsigned int i;
+	struct kfilter *kfilter;
+	const char *skip_funcs[] = {
+		"syscall_trace_enter_phase1",
+		"smp_irq_work_interrupt",
+		"syscall_slow_exit_work",
+	};
+
+	for (i = 0; i < ARRAY_SIZE(skip_funcs); i++) {
+		bool found = false;
+		const char *name = skip_funcs[i];
+		struct kfilter *pos, *tmp;
+
+		/* Don't skip it if user particularly want to see them*/
+		list_for_each_entry_safe(pos, tmp, &kernel->filters, list) {
+			if (strcmp(pos->name, name))
```

Comment:
> Also you made it unconditional, but what if some user really want to see those functions?

@namhyung ,
This patch can handle the case you said.
Because `kernel->filters` and `kernel->patches` users gave are checked in the above for loop
before skip the particular kernel functions that seem to be rarely used.

Reply:
Ah, missed that part, sorry.

But I still want to see all functions when `--kernel-full` option is used.  And could you please change `sys_clock_gettime()` to be part of this change as well?